lib/components/user/nav-login-button-auth0.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { useAuth } from 'use-auth0-hooks'
+
+import NavLoginButton from './nav-login-button'
+
+/**
+ * This component wraps NavLoginButton with Auth0 information.
+ */
+export default function NavLoginButtonAuth0 (props) {
+  const { isAuthenticated, login, logout, user } = useAuth()
+
+  const handleLogin = () => login({ appState: {returnTo: window.location.href} })
+  const handleLogout = () => logout({ returnTo: window.location.href })
+
+  return (
+    <NavLoginButton
+      {...props}
```

Comment:
Let's not spread these props.

Reply:
We still need to forward `id`, `links`, `profile`, `className` and `style` to that component.

Reply:
Yea, I understand. I'm just commenting on the syntax. Instead of using the spread operator (`...props`), I'm asking if you can specify them individually in NavLoginButton's props.